### PR TITLE
zarr data

### PR DESCRIPTION
**Pull request by will-moore**
List of publicly available ome-zarr data, ported from https://github.com/IDR/idr.openmicroscopy.org/pull/108

Current screenshot: at https://github.com/ome/blog/pull/67/commits/2b63cac7b79ecedcf526633ac132b6aef0b018cc

Screenshot 2020-11-09 at 11 04 15

### CODE REVIEW

**Comment by joshmoore** [comment]
Two quick thoughts:
 - the data file should likely have the blog data in its name
 - also, did you want to try the json solution here?

**Comment by will-moore** [comment]
Are we intending to update this list with any `v0.2` data? Do I need to list the 'first_version' available for each file? Probably not if this is just a blog post.
Json data in progress...

**Comment by manics** [comment]
For the final version it'd be nice to link back to an image.sc thread for people to comment

**Comment by joshmoore** [comment]
Built locally. In general, looks awesome. Random things that occur to me that don't necessarily need to be implemented here:
 - study's could be linkable
 - size columns could be sortable
 - table could be filterable
 - as discussed, source code could be local this repo (cc: @manics)
 - snoopy doesn't build these unfortunately (cc: @sbesson)
 - a private image.sc post can be pre-created
 - do we need to explain the versioning of this blog post? (and that plates are coming?)
 - do you have any sample code, @will-moore, for accessing the json?

**Comment by will-moore** [comment]
I would simply link to https://raw.githubusercontent.com/will-moore/blog/zarr_data/_data/zarr_data_2020-11-04.json for the JSON (update branch to ome/master).
I think maybe linking to a study is a distraction? We're not really interested in a whole IDR study here, just individual zarr images?
I don't know that the table is big enough or permanent enough to spend time with sortable, filterable etc?
I do explain some versioning of the zarr data in this post. I can add that HCS is coming. Is something else missing?
We still haven't discussed versioning strategy for the ome-zarr spec yet. I think one version number is better than separate ones for multiscales, labels, HCS etc. e.g HCS and other fixes (e.g. https://github.com/ome/omero-ms-zarr/issues/77) == v0.2.

File: _posts/2020-11-04-zarr-data.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                    <th>Z</th>
+                    <th>C</th>
+                    <th>T</th>
+                    <th title="Link to the data in ome-zarr format">ome-zarr</th>
+                </thead>
+                <tbody>
+                {% for row in site.data.zarr_data_2020-11-04.images %}
+                    <tr>
+                        <td>
+                            <a href="/webclient/?show=image-{{ row.Image }}">
```
Comment (joshmoore) [inline _posts/2020-11-04-zarr-data.md:46]: This will need to be an explicit URL to idr.

File: _posts/2020-11-04-zarr-data.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                                <img style="margin:0" src="http://idr.openmicroscopy.org/webclient/render_thumbnail/{{row.Image}}/"/>
```
Comment (joshmoore) [inline _posts/2020-11-04-zarr-data.md:47]:
```suggestion
                                <img style="margin:0" src="https://idr.openmicroscopy.org/webclient/render_thumbnail/{{row.Image}}/"/>
```
to make @manics happy.

File: _posts/2020-11-04-zarr-data.md (hunk 1 above)
Comment (joshmoore) [reply _posts/2020-11-04-zarr-data.md:46]: (And it should fix travis)

**Review by joshmoore — COMMENTED** [review]
Few minor changes needed. To my list of points:

> I would simply link to https://raw.githubusercontent.com/will-moore/blog/zarr_data/_data/zarr_data_2020-11-04.json for the JSON 

WFM

> I think maybe linking to a study is a distraction? We're not really interested in a whole IDR study here, just individual zarr images?

Here that's certainly fine. When we move back to the IDR page, we can re-review. Perhaps also with general styling updates along with ITR, etc.

> I don't know that the table is big enough or permanent enough to spend time with sortable, filterable etc?

Agreed for here. But eventually...

> I do explain some versioning of the zarr data in this post. I can add that HCS is coming. Is something else missing?

:+1:

> We still haven't discussed versioning strategy for the ome-zarr spec yet. I think one version number is better than separate ones for multiscales, labels, HCS etc. e.g HCS and other fixes (e.g. ome/omero-ms-zarr#77) == v0.2.

Yeah, we really need to work through these issues.

**Comment by will-moore** [comment]
@joshmoore Fixes in - Just check that I've got the JSON link correct (should be working after this is merged).

File: _data/zarr.tsv [hunk 3]
```diff
@@ -0,0 +1,47 @@
+Image	Name	Study	X	Y	Z	C	T	first_version
```
Comment (joshmoore) [inline _data/zarr.tsv:1]: Can this file be deleted?

**Comment by joshmoore** [comment]
Travis is hanging. Merging to get this out.

**Pull request merged**